x-pack/legacy/plugins/epm/server/lib/elasticsearch/template/install.ts

```diff
@@ -63,12 +68,12 @@ async function installTemplate({
   const templateName = generateTemplateName(dataset);
   let pipelineName;
   if (dataset.ingest_pipeline) {
-    pipelineName = getPipelineNameForInstallation(
-      dataset.ingest_pipeline,
+    pipelineName = getPipelineNameForInstallation({
+      pipelineName: dataset.ingest_pipeline,
       dataset,
-      dataset.package,
```

Comment:
Here, positions 3 and 4 appear to be flipped (arg 3 is `packageName`; arg 4 is `datasourceName`) compared to the function definition at https://github.com/elastic/kibana/pull/53848/files#diff-745e02925487aa37c44700d53aa633c9L153